src/main/resources/docs/quick-start.md

```diff
@@ -0,0 +1,532 @@
+6. You will have an option to use HTTPS or SSH - let's use **HTTPS** for now
+
+<strike>7. Select option to **Open in Desktop**</strike>
+
+<strike>8. If you have Sourcetree installed, this will prompt a message to open via Sourcetree, which you can select that option to continue</strike>
+
+</div>
+
+<strike>
+> /info/ <span>**If you'd rather use command line to clone your repo, perform the 
+> following steps instead of steps 5 thru 8 above.** 
+> 1. Open Command Line in Windows; or, Terminal for Mac
+> 2. Clone your repo by entering `git clone https://github.com/accountname/todolist.git`<span class="clipboard" data-clipboard-text='git clone https://github.com/accountname/todolist.git'></span> - swap **accountname** with your GitHub account's name
+> 3. Enter `git checkout master`<span class="clipboard" data-clipboard-text='git checkout master'></span> to checkout the master branch
+> 
+> Having fun using command line? Here is a list of [helpful GIT commands](https://confluence.atlassian.com/bitbucketserver/basic-git-commands-776639767.html). 
+> </span>
+
+</strike>
+
+<a name="configs"></a>
+#### [Open to-do project on your local machine and make some top level configs](#configs)
+
+<div class="ordered-list">
+
+
+1. Open IntelliJ and go to **File > Open** and find the root folder, **todolist**, on your machine <strike>- *note: it's likely under your user directory unless you placed the project somewhere else*</strike>
+2. Double click **todolist** to open 
+3. If you are prompted to **import Gradle**, go ahead and do so
+4. In IntelliJ, click on the **Terminal** tab (located towards the bottom left) **NOT ALWAYS**
+5. In the terminal, type in `chmod +x ./alpas`<span class="clipboard" data-clipboard-text='chmod +x ./alpas'></span> and hit return - this will make Alpas executable
+6. Next, in terminal, type `./alpas init com.todo.list`<span class="clipboard" data-clipboard-text='./alpas init com.todo.list'></span> and hit return - this will name your app
+7. While we are in terminal, type `./alpas help`<span class="clipboard" data-clipboard-text='./alpas help'></span> - this displays a list of commands you can run to find information about your app as well as provide quick actions such as creating new entities 
+8. Before we run the project, let's check to make sure IntelliJ is set to build using Java-9.0.7-zulu - Go to **File > Project Structure, then go to Project and make sure Project SDK has **Java 9** selected**; if it doesn't, follow instructions on how to serve Alpas with IntelliJ in the [Installation documentation](/docs/installation)
+9. Now, find **start.kt** in your project - you can either tap the shift key twice to open quick search or follow the path **todolist > src > kotlin > start.kt**
+10. Right click on **start.kt** and select option to **Run**
+11. Once run is successful, click [http://localhost:8080/](http://localhost:8080/) to view the project in your browser
```

Comment:
@armgitaar This feels like a lot of steps. Maybe we can move steps 4 - 7 out to the previous section.